### DISCUSSION TITLE
トラックボールケースについて

### DISCUSSION THREAD

**Discussion by firebone3** [Yowkees/keyball#326] [Q&A]
ビルドガイドを参照してkeyballをビルドしようとしましたが、
トラックボールケースを出力するための.stlファイルが見つかりませんでした。
もしかしたらキットでのみ提供されていますか？それとも海外でその部品を購入する方法はありますか？

**Comment by Yowkees** [comment]
申し訳ありませんがキットのみでの供給となります。
この部品は白銀ラボのオリジナル部品となります。ご検討よろしくお願いいたします。
